src/app/story/storyrow/storyrow.component.html

```diff
@@ -65,3 +65,10 @@
     </div>
   </div>
 </div>
+<div class="node-siblings-column story-row-column total-time-column" *ngIf="first">
+  <app-totaltimes [ngClass]="'total-times'"
```

Comment:
For long HTML elements like this, the formatting would normally be:
```html
<app-totaltimes
  [ngClass]="'total-times'"
  [totalLeadTime]="totalLeadTime"
  [totalProcessingTime]="totalProcessingTime"
  [totalWaitTime]="totalWaitTime"
>
```

With that being said, I don't think we follow that rule in this code base.